client/flutter/lib/pages/onboarding/location_sharing_page.dart

```diff
@@ -22,6 +23,7 @@ class _LocationSharingPageState extends State<LocationSharingPage> {
   }
 
   void _allowLocationSharing() async {
+    await Location().requestPermission();    
     _complete();
```

Comment:
reminder that in the next PR where we're attaching the location data to a user's push notification registration, we'll want to add jitter so that the location isn't exact ([example here](https://gist.github.com/michiel/7984227))